### ISSUE TITLE
Zooming mode: scrollbars not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#147]
```
The scrollbars can't be moved left-right or up-down in zooming mode. We tested 
this on different Windows machines and MacOSX in the class. Version 1.5.36.
```

Original issue reported on code.google.com by `jkosc...@asu.edu` on 17 May 2014 at 10:59

**Comment by jkoschinsky** [comment]
Verified in GeoDa 1.7.63 Mac OSX
verify in Windows

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.89 (Win 7 Professional (64-bit)).
Zooming works.
